src/Umbraco.Web.UI.Client/src/views/propertyeditors/blocklist/umbblocklistblock.component.js

```diff
@@ -49,19 +49,20 @@
             $scope.index = model.index;
             $scope.parentForm = model.parentForm;
             $scope.valFormManager = model.valFormManager;
+            $scope.view = assetsService.appendCacheBuster(model.view);
 
             if (model.stylesheet) {
                 var shadowRoot = $element[0].attachShadow({ mode: 'open' });
                 shadowRoot.innerHTML = `
                     <style>
-                    @import "${model.stylesheet}"
+                    @import "${assetsService.appendCacheBuster(model.stylesheet)}"
                     </style>
-                    <div ng-include="'${model.view}'"></div>
+                    <div ng-include="'${$scope.view}'"></div>
```

Comment:
This doesn't require escaping the template literal and should be the same as below:
```suggestion
                    <div ng-include="view"></div>
```

Reply:
If these changes are made, the [template literals](https://caniuse.com/template-literals) (backticks) aren't needed anymore.